<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilityStationDialog</class>
 <widget class="QDialog" name="UtilityStationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>588</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>588</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Utility Station</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>400</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>556</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Description</string>
    </attribute>
    <widget class="QLineEdit" name="station_number_edit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>196</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Station Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Operating State</string>
     </property>
    </widget>
    <widget class="QComboBox" name="operating_state_cbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>255</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>16</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Purpose</string>
     </property>
    </widget>
    <widget class="QComboBox" name="purpose_cbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>31</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Station Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="station_type_cbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>75</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>105</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QComboBox" name="location_cbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="height_sbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>165</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height [m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>195</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <widget class="QComboBox" name="network_cbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="station_name_edit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>75</y>
       <width>196</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Station Name</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="installation_year_sbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>3000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>105</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Installation Year</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="capacity_sbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>165</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200000.000000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="capacity_unit_cbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>195</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacity Unit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>285</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of Status Change</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="status_change_date_chbox">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>305</y>
       <width>26</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="status_change_date_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>300</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>station_number_edit</tabstop>
  <tabstop>station_name_edit</tabstop>
  <tabstop>installation_year_sbox</tabstop>
  <tabstop>capacity_sbox</tabstop>
  <tabstop>capacity_unit_cbox</tabstop>
  <tabstop>operating_state_cbox</tabstop>
  <tabstop>status_change_date_chbox</tabstop>
  <tabstop>status_change_date_edit</tabstop>
  <tabstop>purpose_cbox</tabstop>
  <tabstop>station_type_cbox</tabstop>
  <tabstop>location_cbox</tabstop>
  <tabstop>height_sbox</tabstop>
  <tabstop>network_cbox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UtilityStationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UtilityStationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
